lib/node_modules/@stdlib/stats/base/dists/levy/cdf/benchmark/benchmark.js

```diff
@@ -21,28 +21,37 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
+var Float64Array = require( '@stdlib/array/float64' );
+var uniform = require( '@stdlib/random/array/uniform' );
 var randu = require( '@stdlib/random/base/randu' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
-var EPS = require( '@stdlib/constants/float64/eps' );
 var pkg = require( './../package.json' ).name;
 var cdf = require( './../lib' );
 
 
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
-	var scale;
+	var len;
 	var mu;
 	var x;
+	var c;
 	var y;
 	var i;
 
+	len = 100;
+	x = new Float64Array( len );
+	mu = new Float64Array( len );
+	c = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = ( randu() * 20.0 ) - 10.0;
```

Comment:
@AadishJ We can directly draw the random numbers from `uniform` here for better clarity, e.g. `uniform( -10.0, 10.0 )` instead of `( randu() * 20.0 ) - 10.0`.